diff --git a/tests/phpunit/CRM/Core/BAO/FileTest.php b/tests/phpunit/CRM/Core/BAO/FileTest.php
index a56c19f86e..1beac2e510 100644
--- a/tests/phpunit/CRM/Core/BAO/FileTest.php
+++ b/tests/phpunit/CRM/Core/BAO/FileTest.php
@@ -200,7 +200,7 @@ class CRM_Core_BAO_FileTest extends CiviUnitTestCase {
       $invalidFileId ?: $fileId
     );

-    $this->assertEquals($expectedResult, $result);
+    $this->assertEquals($expectedResult, $result, 'genTs is ' . date('Y-m-d H:i:s', $genTs) . ' and now is ' . date('Y-m-d H:i:s'));
   }

 }
